analyzers/tests/SonarAnalyzer.UnitTest/TestCases/SymbolicExecution/Roslyn/HashesShouldHaveUnpredictableSalt.cs

```diff
@@ -202,7 +189,7 @@ public void ByteArrayCases(byte[] passwordBytes)
 
         var shortArray = new byte[] { 0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14 };
         rng.GetBytes(shortArray);
-        new PasswordDeriveBytes(passwordBytes, shortArray);                                         // FIXME Non-compliant
+        new PasswordDeriveBytes(passwordBytes, shortArray);                                         // Noncompliant
```

Comment:
What about adding also a hardcoded array that is longer than 16 bytes for completion?

Reply:
Good idea. I'll add it to the VB test cases as well.

Reply:
Sorry I was not clear here - please add it without passing it after from the random number generator to be sure it raises for hardcoded arrays.

Reply:
Added it (and kept the previous one as well, it's good to cover that too).